### PR TITLE
12.0 mig delivery price rule untaxed

### PR DESCRIPTION
**Pull request by rdeheele**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Warning in runbot:
```
2019-06-14 07:10:21,892 161 WARNING openerp_test odoo.models: delivery.price.rule.create() with unknown fields: standard_price
```

File: delivery_price_rule_untaxed/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# Copyright 2018 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Delivery price rule untaxed",
+    "summary": "Add untaxed amount to variables for price delivery price rule",
+    "version": "12.0.1.0.0",
+    "development_status": "Beta",
+    "category": "Stock",
+    "website": "https://github.com/OCA/delivery-carrier/tree/"
+               "10.0/delivery_price_rule_untaxed",
```
Comment (rousseldenis) [inline delivery_price_rule_untaxed/__manifest__.py]:
```suggestion
               "12.0/delivery_price_rule_untaxed",
```

File: delivery_price_rule_untaxed/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```
Comment (rousseldenis) [inline delivery_price_rule_untaxed/__manifest__.py]: Remove all these. No more necessary

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. Minor comments

File: delivery_price_rule_untaxed/__manifest__.py (hunk 2 above)
Comment (rdeheele) [reply delivery_price_rule_untaxed/__manifest__.py]: thanks, it's done

**Review by hurrinico — APPROVED** [review]
:+1:

**Comment by rousseldenis** [comment]
@rdeheele Could you just squash your commits before merge ?

**Comment by GSLabIt** [comment]
@rousseldenis is this stale?

**Comment by rousseldenis** [comment]
> @rousseldenis is this stale?

Seems no answer from @rdeheele. If you want to supersede, don't hesitate

**Comment by GSLabIt** [comment]
> > @rousseldenis is this stale?
> 
> Seems no answer from @rdeheele. If you want to supersede, don't hesitate

What's the right way to supersede? New pr?

**Comment by rousseldenis** [comment]
> > > @rousseldenis is this stale?
> > 
> > 
> > Seems no answer from @rdeheele. If you want to supersede, don't hesitate
> 
> What's the right way to supersede? New pr?

Yes. The good way is to try taking commits from this too.

**Comment by GSLabIt** [comment]
> > > > @rousseldenis is this stale?
> > > 
> > > 
> > > Seems no answer from @rdeheele. If you want to supersede, don't hesitate
> > 
> > 
> > What's the right way to supersede? New pr?
> 
> Yes. The good way is to try taking commits from this too.

Any guide on how to take commits from this one?

**Comment by rousseldenis** [comment]
@GSLabIt Hehe, not so complicated:

`git remote add rdeheele https://github.com/rdeheele/delivery-carrier.git`

`git fetch rdeheele`

`git checkout <new branch> rdeheele/12.0-mig-delivery_price_rule_untaxed`

`git fetch OCA` (if you have already OCA remote - if not, add it like above)

`git rebase OCA/12.0`

Then, you are ready to finish work and push like other ones. I hope this helps.

**Comment by GSLabIt** [comment]
> @GSLabIt Hehe, not so complicated:
> 
> `git remote add rdeheele https://github.com/rdeheele/delivery-carrier.git`
> 
> `git fetch rdeheele`
> 
> `git checkout <new branch> rdeheele/12.0-mig-delivery_price_rule_untaxed`
> 
> `git fetch OCA` (if you have already OCA remote - if not, add it like above)
> 
> `git rebase OCA/12.0`
> 
> Then, you are ready to finish work and push like other ones. I hope this helps.
> 
> 

Thx

**Comment by rousseldenis** [comment]
Closing this as very old. Supersedes by #408

**Pull request closed without merging**